### PR TITLE
modified script to allow for creation of drafts in _drafts

### PR DESCRIPTION
**Pull request by vsential**
Modified script for creation of drafts in _drafts.

TODO:
- Allow for drafts to be promoted to posts via the script

### CODE REVIEW

File: initpost.sh [hunk 1]
```diff
@@ -43,10 +44,20 @@ FILE_NAME="${CURRENT_DATE}-${POST_NAME}.md"
...
+fi
+
+if [[ "${1}" == "-d" || "${1}" == "--draft" ]]; then
+    DIST_FOLDER="$DRAFTPATH"
+    FILE_NAME="${POST_NAME}.md"
+fi
 
 # Set your blog URL
-BLOG_URL="http://willianjusten.com.br"
+BLOG_URL="http://vsential.com"
```
Comment (willianjusten) [inline initpost.sh:65]: Please, it was my mistake, change this to something like "http://yourblogurl.com", it's more generic than our url...

**Comment by willianjusten** [comment]
I'll wait for the another part to accept this PR. =)

**Comment by vsential** [comment]
Sorry about that!  Changed and thank you!

**Comment by willianjusten** [comment]
Nice =)

**Pull request merged**